.travis.yml

```diff
@@ -728,6 +728,7 @@ script:
     # Build and execute all examples.
     - if [ "${ALPAKA_CI_ANALYSIS}" == "OFF" -a "${ALPAKA_ACC_GPU_CUDA_ONLY_MODE}" == "OFF" ] ;then ./travis/compileExec.sh "example/bufferCopy" ./bufferCopy ;fi
     - if [ "${ALPAKA_CI_ANALYSIS}" == "OFF" -a "${ALPAKA_ACC_GPU_CUDA_ONLY_MODE}" == "OFF" ] ;then ./travis/compileExec.sh "example/helloWorld" ./helloWorld ;fi
+    - if [ "${ALPAKA_CI_ANALYSIS}" == "OFF" -a "${ALPAKA_ACC_GPU_CUDA_ONLY_MODE}" == "OFF" -a ${ALPAKA_CUDA_VER_MAJOR} > 7 -a ${ALPAKA_CUDA_VER_MINOR} > 0] ;then ./travis/compileExec.sh "example/helloWorldLambda" ./helloWorldLambda ;fi    
```

Comment:
This is not correct because it excludes cuda 8.0. (maj>7) || ((maj == 7) && (min >5)) should be better.